routes/api/users.js

```diff
@@ -2,6 +2,8 @@ const mongoose = require("mongoose");
 const router = require("express").Router();
 const passport = require("passport");
 const User = mongoose.model("User");
+const TokenService = require('../../services/TokenService');
```

Comment:
this pascal case naming convention should only hold if `TokenService` is a class

Reply:
I think I could convert this to a class once the refactor to es6 is done.